web/vapour-style.json

```diff
@@ -313,20 +363,45 @@
           "type": "identity",
           "property": "rel_colour"
         },
-        "circle-opacity": 0.5,
+        "circle-opacity": 1,
         "circle-radius": {
           "base": 1,
           "stops": [
             [
               12,
-              12
+              3
             ],
             [
               18,
-              8
+              10
             ]
           ]
-        }
+        },
+        "circle-stroke-width": 1
```

Comment:
I'm not a big fan of this new stop display, as we don't handle the stop position correctly for now.
![screenshot-2018-4-24 http localhost](https://user-images.githubusercontent.com/919962/39207237-c0b8588e-47ff-11e8-879e-3697770db1a6.png)

My idea about the stop_position was to:
* use the stop_position as a dot on the line tracks (just like you did). We could even project the bus stops that does not have any stop_position on the track, so each bus stop have its dot on the line
* use a bus icon for the bus stop platform (maybe a slightly different one that the default display, to make them stand in the front of the others that are not part of the current line)

Reply:
I just add to the TODO list